package.json

```diff
@@ -33,7 +34,8 @@
     "vue-router": "^3.2.0",
     "vue-worker": "^1.2.1",
     "vuetify": "^2.2.29",
-    "vuex": "^3.4.0"
+    "vuex": "^3.4.0",
+    "webpack-bundle-analyzer": "^4.4.0"
```

Comment:
You installed new dependencies here. 

Were new lock files generated and have you tried deleting the existing ones alongside your `node_modules`  then running a fresh installation with `npm install` ?